### ISSUE TITLE
[Bug] Passcode textbox blocked after wrong input

### ISSUE THREAD

**Issue by And96** [standardnotes/desktop#397]
I am using Standard Notes on my notebook with **Passcode lock enabled**. 
if I type a wrong code, I lose focus on the textbox and I am unable to type again the right code.

Version: **Desktop Windows 3.0.16**

![StandardNotes_PasscodeInputBlocked](https://user-images.githubusercontent.com/42185838/65355607-a2b74880-dbf2-11e9-9498-056e7adecbf5.PNG)

**Comment by chanclr** [comment]
I am also encountering this problem on Windows 8.1 Desktop 3.0.16. I can select and delete the passcode, but not type a new one, until switching to another window and back.

**Comment by mobitar** [comment]
Can you click Help > Toggle Error Console, and see if it reports any errors when this happens?

**Comment by chanclr** [comment]
I'm assuming you mean the console tab- no errors there. I have also found out that similar behaviour happens after deleting a note- I can't click on the search box until switching to another window and back. These issues have only come up recently so I'm guessing something to do with the latest update. Clicking one of the menu items also allows me to type again.

**Comment by And96** [comment]
> 
> 
> Can you click Help > Toggle Error Console, and see if it reports any errors when this happens?

I can't find any errors.

I don't if it can be useful but I can tell you that the problem is present only on the desktop version, while on Android and on web version app.standardnotes.org it work fine.

**Comment by mrankine** [comment]
I also find this.

**Comment by chanclr** [comment]
By the way the first time this happened my initial reaction was that the password input had gone somewhere else or was captured by something else. Now I'm thinking it's not _that_ likely since other people have the issue, but if anyone can confirm, it would be appreciated.

**Comment by mrankine** [comment]
I looked in the error console, no errors are reported.

It's not another application stealing focus.

The behaviour is:
- enter an incorrect password
- "invalid passcode" message box pops up, click OK
- passcode field accepts delete key, but does not accept any other input

This has been the case for the past few weeks. (I've also found the search box and note textarea having similar problems, intermittently. But I haven't had a chance to pin that down and replicate yet.)

Windows 10, SN 3.0.16.
